src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx

```diff
@@ -84,201 +84,153 @@ spec:
+
+      * `cloud.provider` ("gcp")
+      * `cloud.platform` ("`gcp_compute_engine`")
+      * `cloud.account.id`
+      * `cloud.region`
+      * `cloud.availability_zone`
+      * `host.id`
+      * `host.image.id`
+      * `host.type`
+
+        ```yaml
+        processors:
+        resourcedetection:
+            detectors: [gke, gce]
+        ```
+      </Collapser>
+      <Collapser
+          id="attributes-general"
+          title="Configure the Kubernetes Attributes processor (general)"
+      >
+
+      When we run the `k8sattributes` processor as part of the OpenTelemetry Collector running as an agent, it detects the IP addresses of pods sending telemetry data to the OpenTelemetry Collector agent, using them to extract pod metadata. Below is a basic Kubernetes manifest example with only a processors section. To deploy the OpenTelemetry Collector as a `DaemonSet`, read this [comprehensive manifest example](https://github.com/newrelic-forks/microservices-demo/tree/main/src/otel-collector-agent).
 
-  <Step>
-    ### Configure the resource detection processor [#resource-detection]
+      ```yaml
+      processors:
+      k8sattributes:
+          auth_type: "serviceAccount"
+          passthrough: false
+          filter:
+          node_from_env_var: KUBE_NODE_NAME
+          extract:
+          metadata:
+              - k8s.pod.name
+              - k8s.pod.uid
+              - k8s.deployment.name
+              - k8s.cluster.name
+              - k8s.namespace.name
+              - k8s.node.name
+              - k8s.pod.start_time
+          pod_association:
+          - from: resource_attribute
+              name: k8s.pod.uid
+      ```
+    </Collapser>
+      <Collapser
+          id="rbac"
+          title="Configure the Kubernetes Attributes processor (RBAC)"
+      >
 
-    The `resourcedetection` processor gets host-specific information to add additional context to the telemetry data being processed through the collector. In this example, we use Google Kubernetes Engine (GKE) and Google Compute Engine (GCE) to get Google Cloud-specific metadata, including:
+      You need to add configurations for role-based access control (RBAC). The `k8sattributes` processor needs `get`, `watch`, and `list` permissions for pods and namespaces resources included in the configured filters. This [example](https://github.com/newrelic-forks/microservices-demo/blob/main/otel-kubernetes-manifests/otel-collector-agent.yaml#L43-L69) shows how to configure role-based access control (RBAC) for `ClusterRole` to give a `ServiceAccount` the necessary permissions for all pods and namespaces in the cluster.
+      </Collapser>
+      <Collapser
+          id="discovery-filter"
+          title="Configure the Kubernetes Attributes processor (discovery filter)"
```

Comment:
lowercase "attributes" on all of these collapser titles

Reply:
Nice catch!